### PR TITLE
readme

### PR DESCRIPTION
**Pull request by hsy3418**
# Description

Create a readme and move the list of the security control links to a policy inventory readme.

## Type Of Change

To help us figure out who should review this PR, please put an X in all the areas that this PR affects.

- [X] Docs
- [ ] Installation
- [ ] Performance and Scalability
- [ ] Security
- [ ] Test and/or Release
- [ ] User Experience

## Issue Ref (Optional)

Which issue(s) this PR fixes (optional, using fixes #<issue number>(, fixes #<issue_number>, ...) format, will close the issue(s) when the PR gets merged): Fixes #

## Notes

Add special notes for your reviewer here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,85 @@
 # Security Policies
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fraspbernetes%2Fk8s-security-policies.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fraspbernetes%2Fk8s-security-policies?ref=badge_shield)
 
+## Introduction
+This repository provides a security policies library that is used for securing Kubernetes clusters configurations. The security controls is created based on [CIS Kubernetes benchmark](https://cloud.google.com/kubernetes-engine/docs/concepts/cis-benchmarks) ,and rules defined in [Kubesce.io](https://kubesec.io/).
```
Comment (xUnholy) [inline README.md:5]: Typo

```suggestion
This repository provides a security policies library that is used for securing Kubernetes clusters configurations. The security controls is created based on [CIS Kubernetes benchmark](https://cloud.google.com/kubernetes-engine/docs/concepts/cis-benchmarks) ,and rules defined in [Kubesec.io](https://kubesec.io/). 
```

**Review by xUnholy — COMMENTED** [review]
Just need to do a small proof read and adjust any typos otherwise looks great!

**Review by xUnholy — APPROVED** [review]
We should investigate if it's possible to have our docs point to this readme remotely so we can keep the docs in their respective repos rather than duplicating it in both repos.

**Comment by hsy3418** [comment]
right @xUnholy let me check if it's possible, otherwise we can put a link in this repo of the doc website?

**Pull request merged**